src/css/showHide.js

```diff
@@ -4,6 +4,23 @@ define( [
 	"../css/var/isHidden"
 ], function( jQuery, dataPriv, isHidden ) {
 
+var defaultDisplayMap = {};
+
+function setDefaultDisplay( elem ) {
```

Comment:
That's because of the `if none the block` thing? So if there would be a case, when, say inline element would be added to the hidden body, user could re-define it.

I like it, i would even consider it to be a blocker for the release